tracker.py

```diff
@@ -88,10 +88,20 @@ def http_scraper(self, torrent, tracker):
         try:
             answer_tracker = requests.get(tracker, params=params, timeout=5)
             list_peers = bdecode(answer_tracker.content)
-
-            for p in list_peers['peers']:
-                s = SockAddr(p['ip'], p['port'])
-                self.dict_sock_addr[s.__hash__()] = s
+            offset=0
+            if not type(list_peers['peers']) == dict:
+                    for _ in range(len(list_peers['peers'])//6):
+                        ip  =   struct.unpack_from("!i", list_peers['peers'], offset)[0]
+                        ip =   socket.inet_ntoa(struct.pack("!i", ip))
+                        offset  +=  4
+                        port   =   struct.unpack_from("!H",list_peers['peers'], offset)[0]
+                        offset  +=  2
+                        s=SockAddr(ip,port)
+                        self.dict_sock_addr[s.__hash__()]=s
```

Comment:
Can you please use only one space between the "=" ? Like :
`ip = struct....`

Reply:
If you could also add some comments on why you added this part of the code it would be awesome

Reply:
Fixed the spacing and I have included the comments. Any more improvements or changes which you want me to do are welcomed.